command_parse.h

```diff
@@ -65,6 +65,9 @@ enum CommandResult parse_unsubscribe     (struct Buffer *buf, struct Buffer *s,
 enum CommandResult parse_unsubscribe_from(struct Buffer *buf, struct Buffer *s, intptr_t data, struct Buffer *err);
 #endif
 
+enum CommandResult mutt_parse_rc_line_cwd(const char *line, struct Buffer *err, char *cwd);
```

Comment:
eh should the arg order be re-ordered to put `err` at the end?

Reply:
yes please